### PR TITLE
Fix/vue query unref query params

### PR DESCRIPTION
**Pull request by anymaniax**
## Status

**READY**

## Description

#1711

### CODE REVIEW

**Review by soartec-lab — DISMISSED** [review]
That's nice! thank you!

File: tests/configs/zod.config.ts [hunk 1]
```diff
@@ -134,7 +134,7 @@ export default defineConfig({
       client: 'zod',
     },
     input: {
-      target: '../specifications/additional-properties.yaml',
+      target: '../specifications/translation.yaml',
```
Comment (soartec-lab) [inline tests/configs/zod.config.ts:137]: @anymaniax 

Is this an intended change?

Comment (anymaniax) [reply tests/configs/zod.config.ts:137]: Yes the other file doesn’t exist and this one has also additional properties

**Review by melloware — DISMISSED** [review]

File: tests/configs/zod.config.ts (hunk 1 above)
Comment (soartec-lab) [reply tests/configs/zod.config.ts:137]: Aha, I noticed that `specifications/additional-properties.yaml` is a non-existent file..

**Review by soartec-lab — APPROVED** [review]
It's great!

**Comment by soartec-lab** [comment]
Oh, there was a conflict.

@anymaniax 
Could you please fix this?

**Review by melloware — APPROVED** [review]

**Comment by anymaniax** [comment]
it's done

**Review by melloware — APPROVED** [review]

**Review by soartec-lab — APPROVED** [review]

**Pull request merged**